tests/datalab/issue_manager/test_label.py

```diff
@@ -103,3 +103,10 @@ def test_get_summary_parameters(self, issue_manager, monkeypatch):
         summary_parameters.pop("pred_probs")
         expected_parameters.pop("pred_probs")
         assert summary_parameters == expected_parameters
+
+    def test_find_issues_low_memory(self, pred_probs, lab):
+        clean_learning_kwargs = {"low_memory": True}
+        issue_manager_lm = LabelIssueManager(
+            datalab=lab, clean_learning_kwargs=clean_learning_kwargs
+        )
+        issue_manager_lm.find_issues(pred_probs=pred_probs)
```

Comment:
IMO this block should instead be added at the end of an existing unit test that does the find_issues without low_memory, and then you should finally check the jaccard similarity between the 2 runs.